main.go

```diff
@@ -25,15 +24,25 @@ var (
 	version = "head"
 )
 
+type DockerSecret struct {
+  Value string
+}
+
+func (s *DockerSecret) UnmarshalText(b []byte) error {
+  v, err := os.ReadFile(string(b))
```

Comment:
Would it be cleaner if you trim and remove spaces here? I think all Docker secrets would want that.

Reply:
If there is a space in the password, it will be deleted.

Reply:
Are you sure? https://yourbasic.org/golang/trim-whitespace-from-string/ only spaces beginning and end would be removed. Not in the middle.

Reply:
Well, once again, if a space in the password is at the beginning or at the end, then it will be deleted

Reply:
I think that's ok, no? If you don't think so then we should just not strip any spaces. I don't know enough about secrets to know what Docker does.

Reply:
I know how that works. What I am not sure about different operating system.  For example, in docker doc it says `printf "This is a secret" | docker secret create my_secret_data -` What if they do `"This is a secret "`? 

But I think you are right. No reason to trim it.